packages/nvidia_nat_openpipe_art/src/nat/plugins/openpipe/trainer.py

```
@@ -0,0 +1,659 @@
+            run_id: The run identifier
+
+        Returns:
+            dict: Metrics from the training run
+        """
+        metrics = {"run_id": run_id, "total_epochs": len(self._job_refs), "jobs": []}
+
+        for job_ref in self._job_refs:
+            try:
+                status = await self.trainer_adapter.status(job_ref)
+                metrics["jobs"].append({"job_ref": job_ref.model_dump(), "status": status.model_dump()})
+            except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)